api.yaml

```diff
@@ -139,185 +156,77 @@ info:
 
     Need more examples? Play with [**our mock server**](https://github.com/Tiqets/supplier-api/tree/master/supplier_server_mock) to see some examples of expected behavior.
 
-    Requires Python to be installed on your development environment.
+    Python must be installed on your development environment.
 
     [![mock server screenshot](https://raw.githubusercontent.com/Tiqets/supplier-api/master/docs/mockserver.png)](https://github.com/Tiqets/supplier-api/tree/master/supplier_server_mock)
 
   contact:
+    name: "Tiqets API Support"
     email: apisupport@tiqets.com
-  version: 1.6.0
+    version: 2.0.0
 tags:
   - name: Availability
```

Comment:
The availability header appears two times in a row, maybe we should move this description into an endpoint description.

![image](https://user-images.githubusercontent.com/97532/197634644-774b42df-ae9e-444c-85c5-46b3b8130c54.png)